api/openRoutes.ts

```diff
@@ -8,6 +8,7 @@ import getLovRefDataRouter from './prd/lov';
 // TODO: rename from prdRouter
 import getappInsightsInstrumentationKey from './monitoring-tools';
 import prdRouter from './register-org';
+import registerOrgRouter from './registerOrganisation';
 import addressRouter from './addresses';
```

Comment:
ordering

Reply:
addresses alphabetically before others. However this is openRoutes so actually not necessary